modules/nw-networkpolicy-audit-enable.adoc

```diff
@@ -0,0 +1,67 @@
+[id="nw-networkpolicy-audit-enable_{context}"]
+= Enabling network policy audit logging
+
+As a cluster administrator, you can enable network policy audit logging for a namespace.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. To enable network policy audit logging for a namespace, enter the following command:
++
+[source,terminal]
+----
+$ oc annotate namespace <namespace> \
+  k8s.ovn.org/acl-logging='{ "deny": "alert", "allow": "notice" }'
+----
++
+--
+where:
+
+`<namespace>`:: Specifies the name of the namespace.
+--
++
+[TIP]
+====
+You can alternatively apply the following YAML to enable audit logging:
+
+[source,yaml]
+----
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: <namespace>
+  annotations:
+    k8s.ovn.org/acl-logging: |-
+      {
+        "deny": "alert",
+        "allow": "notice"
+      }
+----
+====
++
+.Example output
+[source,terminal]
+----
+namespace/verify-audit-logging annotated
+----
+
+.Verification
+
+. Output the latest entries in the network policy audit log:
```

Comment:
Nit: Is "display" or "view" better than "output" for the verb?

In the `for` loop, how do you feel about using `for pod` instead of `for node`.  This ignoramus was a little unsure for a few minutes whether the log was on the node or in the pod.

Reply:
If we give lot more details in the document it could be further added logging is on ovnkube-node pod 's ovn-acl-logging container (/var/log/ovn/acl-audit-log). Also the logs end up on ovnkube-node pod on the node on which the pod that has ACL applied and accessed is scheduled.

Reply:
@asood-rh, that's a fantastic idea, but I might open a fix-up PR to include that so the bulk if this is merged in time for 4.8.

Reply:
@asood-rh, I've created https://github.com/openshift/openshift-docs/pull/33859 for this.